### ISSUE TITLE
AbstractCompiler don't work with class declared with ::class

### ISSUE THREAD

**Issue by syrm** [K-Phoen/rulerz#58]
If you declare your traits like that : 
```php
protected function getExecutorTraits()
    {
        return [
            FilterTrait::class,
            FilterBasedSatisfaction::class
        ];
    }
```

AbstractCompiler compile wrong class.
Ligne 25 should be replaced from :
```php
            return "\t" . 'use ' . $trait . ';';
```
to : 
```php
            return "\t" . 'use \\' . ltrim($trait, '\\') . ';';
```

**Comment by K-Phoen** [comment]
Do you want to make a PR? I'd be happy to review it :)

**Comment by K-Phoen** [comment]
Fixed by #59
